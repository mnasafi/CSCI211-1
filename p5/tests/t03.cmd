2 0 tests/t03.in results/t03.myout
