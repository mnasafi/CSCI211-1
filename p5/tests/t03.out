1: Homer entered store
2: Marge entered store
3: Bart entered store
4: Maggie entered store
5: Lisa entered store
7: Lisa done shopping
7: Lisa started checkout with checker 0
8: Maggie done shopping
8: Maggie started checkout with checker 1
9: Bart done shopping
10: Marge done shopping
10: Maggie paid $6 for 2 items to checker 1
10: Bart started checkout with checker 1
11: Homer done shopping
11: Lisa stole $100 and 1 item from checker 0
11: Marge started checkout with checker 0
13: Bart paid $9 for 3 items to checker 1
13: Homer started checkout with checker 1
15: Marge paid $12 for 4 items to checker 0
18: Homer paid $15 for 5 items to checker 1
registers[0] = $12
registers[1] = $130
time = 19
